mysql-test/t/threads_running_gtid-master.opt

```diff
@@ -0,0 +1 @@
+--gtid-mode=ON --enforce_gtid_consistency=ON --log_slave_updates=ON
```

Comment:
And this one to rpl_gtid suite?

Reply:
Please move to `mysql-test/suite/rpl_gtid/t/threads_running-master.opt` and remove `--gtid-mode=ON --enforce_gtid_consistency=ON` options from this file.